lib/pfconfig/namespaces/resource/UriFilters.pm

```diff
@@ -1,44 +1,33 @@
-package pfconfig::namespaces::resource::Profile_Filters;
-
+package pfconfig::namespaces::resource::UriFilters;
 =head1 NAME
 
-pfconfig::namespaces::resource::Profile_Filters
+pfconfig::namespaces::resource::UriFilters
```

Comment:
It should still be URI even with underscore IMO

So the standard would be : 
TurkeyFilter
TurkeySandwichFilter
TS_Filter (where TS is the abreviation of Turkey Sandwich)

Would that be it ?